pkgs/os-specific/linux/fbterm/default.nix

```diff
@@ -51,7 +46,6 @@ stdenv.mkDerivation {
   ];
```

Comment:
Would be also nice to move patches after sources.

Reply:
Mostly already done in #155226